core/src/main/resources/filodb-defaults.conf

```diff
@@ -174,6 +174,12 @@ filodb {
     # Enable faster rate/increase/delta calculations. Depends on drop detection in chunks (detectDrops=true)
     faster-rate = true
 
+    # Time ranges for which additional join/by keys need to be added implicitly. Add as a 2D array, example:
+    # [
+    #   [1600224662, 1600229662],
```

Comment:
Do we need a start time and end time. can we not just have one setting for all queries which says for any binary join query with a group by and ignoring older than this time stamp we will apply extra label joins.

Reply:
This config is more flexible. To achieve that goal simply say `[0, someTime]` in the config.